vpr/src/route/rr_graph.cpp

```diff
@@ -865,7 +877,38 @@ static void load_rr_switch_inf(const int num_arch_switches, const float R_minW_n
         }
     }
 }
+t_rr_switch_inf create_rr_switch_from_arch_switch(int arch_switch_idx,
```

Comment:
Add code comments

/* This function creates a routing switch for the usage of routing resource graph, based on a routing switch defined in architecture file.
 *  Since users can specify a routing switch whose buffer size is automatically tuned for routing architecture, the function here sets a definite buffer size, as required by placers and routers.
 */